src/Symfony/Component/DependencyInjection/Tests/ServiceLocatorTest.php

```diff
@@ -18,9 +18,14 @@
 use Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException;
 use Symfony\Component\DependencyInjection\ServiceLocator;
 use Symfony\Contracts\Service\ServiceSubscriberInterface;
-use Symfony\Contracts\Service\Test\ServiceLocatorTest as BaseServiceLocatorTest;
+use Symfony\Contracts\Service\Test\ServiceLocatorTest as LegacyServiceLocatorTestCase;
+use Symfony\Contracts\Service\Test\ServiceLocatorTestCase as BaseServiceLocatorTestCase;
```

Comment:
no need to alias this as `ServiceLocatorTestCase` does not conflict with the class name of the current class